modules/schematics/src/effect/index.spec.ts

```diff
@@ -50,6 +50,44 @@ describe('Effect Schematic', () => {
     ).toBeGreaterThanOrEqual(0);
   });
 
+  it('should create an effect class which has a "loadFoos$" property if feature is set', () => {
+    const options = { ...defaultOptions, feature: true };
+
+    const tree = schematicRunner.runSchematic('effect', options, appTree);
+    const content = tree.readContent(
+      `${projectPath}/src/app/foo/foo.effects.ts`
+    );
+    expect(content).toMatch(
+      /import { Actions, Effect, ofType } from '@ngrx\/effects';/
+    );
+    expect(content).toMatch(
+      /import { LoadFoos, FooActionTypes } from '\.\/foo.actions';/
+    );
+    expect(content).toMatch(/export class FooEffects/);
+    expect(content).toMatch(
+      /loadFoos\$ = this\.actions\$.pipe\(ofType<LoadFoos>\(FooActionTypes\.LoadFoos\)\);/
+    );
+  });
+
+  it('should create an effect class which does not have a "loadFoos$" property if root is set', () => {
```

Comment:
This should be `should create an effect that does not define a source of actions within the root`.
And maybe move the test a bit down? (after the default tests)